### PR TITLE
chromium_src cleanup

### PR DESCRIPTION
**Pull request by bridiver**
still need to run tests

### CODE REVIEW

File: atom/browser/importer/profile_writer.cc [hunk 1]
```diff
@@ -1,114 +1,49 @@
...
-  return folder_name;
-}
-
-// Shows the bookmarks toolbar.
-void ShowBookmarkBar(Profile* profile) {
-  //profile->GetPrefs()->SetBoolean(bookmarks::prefs::kShowBookmarkBar, true);
-}
+namespace importer {
+void ShowImportLockDialog(gfx::NativeWindow parent,
+                          const base::Callback<void(bool)>& callback) {}
```
Comment (darkdh) [inline atom/browser/importer/profile_writer.cc]: @bridiver I think this should call `ProfileWriter::ShowWarningDialog` to emit event.

File: atom/browser/importer/profile_writer.h [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      const base::string16& top_level_folder_name) override;
+  void AddFavicons(const favicon_base::FaviconUsageDataList& favicons) override;
+
+  // Adds the imported autofill entries to the autofill database.
+  void AddAutofillFormDataEntries(
+      const std::vector<autofill::AutofillEntry>& autofill_entries) override;
+  virtual void AddCookies(const std::vector<ImportedCookieEntry>& cookies);
+  void Initialize(atom::api::Importer* importer);
+
+  bool ShowWarningDialog();
```
Comment (darkdh) [inline atom/browser/importer/profile_writer.h]: We might need a static version of `ProfileWriter::ShowWarningDialog` for `ShowImportLockDialog` to call per comment in profile_writer.cc

File: atom/browser/importer/profile_writer.cc (hunk 1 above)
Comment (darkdh) [inline atom/browser/importer/profile_writer.cc]: It was calling here https://github.com/brave/muon/pull/188/files#diff-781cd52d5d2ddb9a36261ffdf1606475L156

File: atom/browser/importer/external_process_importer_client.h [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+      const std::vector<ImportedCookieEntry>&
+          cookies_group);
+
+ private:
+  ~ExternalProcessImporterClient() override;
+
+  // Total number of cookies to import.
+  size_t total_cookies_count_;
+
+  scoped_refptr<InProcessImporterBridge> bridge_;
```
Comment (darkdh) [inline atom/browser/importer/external_process_importer_client.h]: could we add friend class in `::ExternalProcessImporterClient` and not copy bridge here?

File: atom/browser/importer/external_process_importer_client.h [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+  std::vector<ImportedCookieEntry> cookies_;
+
+  // True if import process has been cancelled.
+  bool cancelled_;
```
Comment (darkdh) [inline atom/browser/importer/external_process_importer_client.h]: ditto

File: atom/browser/importer/in_process_importer_bridge.h [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+                       importer::VisitSource visit_source) override;
+
+  void SetKeywords(
+      const std::vector<importer::SearchEngineInfo>& search_engines,
+      bool unique_on_host_and_path) override;
+
+  void SetFirefoxSearchEnginesXMLData(
+      const std::vector<std::string>& search_engine_data) override;
+
+  virtual void SetCookies(const std::vector<ImportedCookieEntry>& cookies);
```
Comment (darkdh) [inline atom/browser/importer/in_process_importer_bridge.h]: virtual keyword can be removed here?

**Comment by darkdh** [comment]
other changes looks good to me.

File: atom/browser/importer/profile_writer.cc (hunk 1 above)
Comment (bridiver) [inline atom/browser/importer/profile_writer.cc]: it still calls that in `ProfileWriter::ShowWarningDialog()`

Comment (darkdh) [inline atom/browser/importer/profile_writer.cc]: ProfileWriter::ShowWarningDialog() is our thing. Chrome‘s version doesn't call it. So we have to call it here. Or the warning dialog won't work on browser-laptop.

**Comment by bridiver** [comment]
this is already merged as part of another commit

**Pull request closed without merging**